.github/workflows/_build.yml

```diff
@@ -256,3 +256,107 @@ jobs:
           git add app/screenshots/*
           git diff-index --quiet HEAD || git commit -m "Updated snapshots for '${{ matrix.variant }}'"
           git push
+
+  Screenshot:
+    name: Screenshots recording
+    runs-on: macos-latest
+    needs: CheckCommitMessage
+    if: ${{ needs.CheckCommitMessage.outputs.head-commit-message == 'Record screenshots' }}
+    permissions:
+      pull-requests: write
+      contents: write
+    timeout-minutes: 30
+
+    strategy:
+      fail-fast: true
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+          cache: gradle
+
+      - name: AVD cache
+        uses: actions/cache@v3
+        id: avd-cache
+        with:
+          path: |
+            ~/.android/avd/*
+            ~/.android/adb*
+          key: avd-${{ env.api-level }}
```

Comment:
as this AVD will use a different api level to the other ones you'll need to use a different key here otherwise the cache is useless and it'll cache the wrong emulator